bootstrap/terraform/main.tf

```diff
@@ -180,6 +174,26 @@ module "eks_blueprints_addons" {
   depends_on = [module.eks.eks_managed_node_groups]
 }
 
+#---------------------------------------------------------------
+# Gatekeeper
+#---------------------------------------------------------------
+module "gatekeeper" {
+  source  = "aws-ia/eks-blueprints-addon/aws"
+  version = "1.1.1"
```

Comment:
If this works but the addon doesn't, can you file an issue on the addon repo and link here?

Reply:
I noticed the Gatekeeper deployment's liveness probe doesn't have an initialDelaySeconds set, or it's too low. This can be a problem when not all cluster pods (like coredns, vpc-cpi, etc.) are up and running right away. If Gatekeeper starts too early, it doesn’t retry enough times and fail.

To work around this, I added a dependency, forcing Gatekeeper to wait until all the addons are up and running, which seems to work now. I will see if I can open a PR to adjust this in the addon repo.